showcase.json

```diff
@@ -3690,5 +3690,27 @@
     "github": "https://github.com/elna-ai",
     "twitter": "https://twitter.com/elna_live",
     "submittableId": "44027684"
+  },
+  {
+    "id": "vaultbet",
+    "name": "VaultBet",
+    "oneLiner": "Redefining online betting on the Internet Computer.",
+    "website": "https://vault-bet.com",
+    "tags": [
+      "Games",
+      "DeFi",
+      "Bitcoin",
```

Comment:
Can you please confirm how you include Bitcoin, Ethereum, and NFTs? I was unable to find features related to these on the platform.

Reply:
Hello, yes, users of our website are able to play our arcade games with ICP, ckBTC and ckETH as well. We also did a crowdfunding campaign where our supporters were awarded NFTs that granted them a number of rewards such as the ability to claim monthly VBT (our own token) which can be used to play on our site as well. If you go to the arcade section on our site you will have the option to deposit and use ICP, ckBTC, ckETH, and VBT for playing. To claim VBT you must connect to our site with a wallet holding at least one of our Market Maker or Pro Player NFTs.